types/defaults-deep/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for defaults-deep 0.2
+// Project: https://github.com/jonschlinkert/defaults-deep
+// Definitions by: Hugo Alliaume <https://github.com/Kocal>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Obj {
+    [k: string]: any;
+}
+
+export default function defaultDeep(obj?: Obj, ...objs: Obj[]): Obj;
```

Comment:
Thanks for the review, I thought it was okay to use `export default func` since `module.exports = func` is like a default export in commonJS (or am I wrong?)

Also I tried to use `export = defaultsDeep;` (from `module-function.d.ts`) but it does not seems to work, I'm having the following error:
> Module '"/home/kocal/Dev/DefinitelyTyped/types/defaults-deep/index"' has no default export.

Reply:
Oh okay, didn't see the comment, I'm now using `import defaults = require('defaults-deep')` and it seems working

Reply:
Yep looks good now.

If you were wondering why the default export doesn't work with CommonJS modules, here is the reason. In an ES6 module, the export default statement is just the exact same as a normal export with the name `default`. If a module `m` has a default export, then someone else consuming this package in CommonJS would use that by doing the equivalent of `require('m').default`.

Reply:
Thanks for the explanation! :)